app/org/sagebionetworks/bridge/json/DateUtils.java

```diff
@@ -158,4 +158,25 @@ public static Long convertToMillisFromDuration(String duration) {
         return null;
     }
 
+    /**
+     * Convert at 8601 time zone offset string (e.g. "-07:00") to a DateTimeZone 
+     * object. 
+     * @param offsetString
+     * @return
+     */
+    public static DateTimeZone parseZoneFromOffsetString(String offsetString) {
+        DateTimeZone zone = null;
+        if (StringUtils.isNotBlank(offsetString)) {
+            try {
+                String[] zoneParts = offsetString.split(":");
+                int hours = Integer.parseInt(zoneParts[0]);
+                int minutes = Integer.parseInt(zoneParts[1]);
+                zone = DateTimeZone.forOffsetHoursMinutes(hours, minutes);
+            } catch(Throwable throwable) {
```

Comment:
We don't want to catch Throwable, since that includes things like ThreadDeathError and OutOfMemoryError. You should catch RuntimeException instead.